pkg/api/v1alpha1/cluster_types.go

```diff
@@ -41,6 +41,9 @@ const (
 	// cluster object.
 	managementAnnotation = "anywhere.eks.amazonaws.com/managed-by"
 
+	// EKSAManagementComponentVersion is an annotation applied to an EKS-A management cluster pointing to the current version of EKS management components.
+	EKSAManagementComponentVersion = "anywhere.eks.amazonaws.com/eksa-management-component-version"
```

Comment:
looking at other field names, is it more consistent to name it "ManagementComponentsVersionAnnotation"

Reply:
Ah, good catch. I had meant to have the`Annotation` suffix there